packages/components/psammead-brand/src/index.jsx

```diff
@@ -274,10 +265,8 @@ Brand.propTypes = {
   borderBottom: bool,
   scriptLink: node,
   skipLink: node,
-  focusRef: oneOfType([
-    func,
-    shape({ current: instanceOf(HTMLAnchorElement) }),
-  ]),
+  // eslint-disable-next-line react/forbid-prop-types
+  focusRef: any,
```

Comment:
Maybe make the prop type a bit more specific, this checks it has the shape of ref (has the current prop) but doesn't bother to check the type of the item referenced (good enough imo)
```suggestion
    focusRef: oneOfType([
    func,
    shape({ current: any }),
  ]),
```